brave/brave-ios
# Fix #7422: Update playlist icons & url bar interactions

## Summary of Changes

This pull request fixes #7422 

Also updates all mentions of "Folder" with "Playlist" in the current version of Playlist

## Submitter Checklist:


## Test Plan:

- New Popover when adding initially:
  - Verify tapping on the popover itself opens playlist
  - Verify tapping Change dismisses the popup and shows the change playlist UI
  - Verify after 4s the popover automatically dismisses
- Tapping on the icon after adding:
  - Verify remove/undo
  - Verify change playlist shows the change playlist UI
    - Verify changing the folder updates the text shown in the menu
  - Verify opening in playlist resumes at the current time in the web page

## Screenshots:

| | | |
| --- | --- | --- |
| ![Simulator Screenshot - iPhone 14 Pro - 2023-05-09 at 16 25 33](https://github.com/brave/brave-ios/assets/529104/d5e90683-f46d-4807-a86f-0238137fe9c0) | ![Simulator Screenshot - iPhone 14 Pro - 2023-05-09 at 16 26 39](https://github.com/brave/brave-ios/assets/529104/f43fc9ae-215a-427f-bd59-30c445abaaec) | ![Simulator Screenshot - iPhone 14 Pro - 2023-05-09 at 16 26 46](https://github.com/brave/brave-ios/assets/529104/a2be487d-5b98-48d9-82e6-6d5865d4df53) |
| ![Simulator Screenshot - iPhone 14 Pro - 2023-05-09 at 16 26 52](https://github.com/brave/brave-ios/assets/529104/9deda575-102c-469d-b3af-ccac5a8bda62) | ![Simulator Screenshot - iPhone 14 Pro - 2023-05-09 at 16 26 35](https://github.com/brave/brave-ios/assets/529104/6b255dfc-400e-4929-bec1-6557df92608e) | ![Simulator Screenshot - iPhone 14 Pro - 2023-05-09 at 16 27 00](https://github.com/brave/brave-ios/assets/529104/ba3564d7-572f-4132-8301-61d732669185) |


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
